GbengaOshinaga/ChatApp
# SIGN UP SCREEN

### What does this PR do?
>- Adds sign up screen and functionality to the application
### Description of Task to be completed?
>- Create a signup screen
>- Integrate sign up with redux
>- Write tests
### How should this be manually tested?
>- Clone the branch
>- Run `yarn install`, then `yarn start`
>- Scan the QR code that appears on the terminal with Expo app on your phone
### Any background context you want to provide?
>- N/A
### What are the relevant pivotal tracker stories?
>- N/A
### Screenshots (if appropriate)
>- N/A
### Questions:
>- N/A
